src/main/java/org/terasology/module/inventory/ui/InventoryGrid.java

```diff
@@ -39,8 +39,8 @@ public class InventoryGrid extends CoreWidget {
     /**
      * Defines the first inventory slot index to which an InventoryGrid should refer to.
      *
-     * Example: If an inventory has 20 slots and cellOffset is set to 5, 15 cells will be drawn starting from slot no. 5 (starting from zero).
-     * Other slots will be not accessible by this InventoryGrid.
+     * Example: If an inventory has 20 slots and cellOffset is set to 5, 15 cells will be drawn starting from slot no. 5 (starting from
+     * zero). Other slots will be not accessible by this InventoryGrid.
```

Comment:
Consider breaking lines at more readable places, e.g.
```
     * Example: If an inventory has 20 slots and cellOffset is set to 5, 15 cells will be drawn 
     * starting from slot no. 5 (starting from zero). Other slots will be not accessible by this InventoryGrid.
```

Reply:
this is standard text, and i do love the line breaks at the end of the line, not arbitrary positions.

Reply:
it's not arbitrary but based on read flow :wink: